src/pages/api/loadTopics.ts

```diff
@@ -9,11 +10,14 @@ export interface LoadTopicsParams {
 
 export interface LoadTopicsResults {
   topics: Topic[]
+  currentTopic: ID | null
 }
 
 export default apiController<LoadTopicsParams, LoadTopicsResults>(async ({ cid }) => {
```

Comment:
It seems like this api is duplicate of the one above. We should remove one of them if possible. Let's work on this later.